File: src/codes/Users_Manual/controller_with_parallel.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::builder([AUTD3::new(Point3::origin())]).open(autd3::link::Nop::builder())?;
# let m = Static::new();
# let g = Null::new();
autd.send((m, g).with_parallel_threshold(Some(0)))?;
# Ok(())
# }

File: src/codes/Users_Manual/controller_with_parallel.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder({autd3::AUTD3(autd3::Point3::origin())}).open(autd3::link::Nop::builder());
//~autd3::Static m;
//~autd3::Null g;
autd.send((m, g).with_parallel_threshold(0));
//~return 0; }